package.json

```diff
@@ -1,68 +1,68 @@
-		"test": "test"
-	},
-	"engines": {
-		"node": ">=10"
-	},
-	"types": "src/cross-cat.d.ts",
-	"scripts": {
-		"build": "tsc",
-		"build:docs": "typedoc",
-		"coverage:cobertura": "nyc --reporter cobertura npm run test:ts:mocha",
-		"coverage:lcov": "nyc npm run test:ts:mocha",
-		"coverage:codacy": "npm run coverage:lcov && npx cross-cat coverage/lcov.info | codacy-coverage",
-		"test:js:junit": "npm run build && mocha test --reporter mocha-junit-reporter",
-		"test:js:mocha": "npm run build && mocha",
-		"test:ts:mocha": "npm run build && mocha --require ts-node/register test/test.ts",
-		"abc": "npx chirpchirp send --message \"abc abc\n abc abc\""
-	},
-	"repository": "github:cpuabuse/cross-cat",
-	"keywords": [
-		"cat",
-		"cli",
-		"concat"
-	],
-	"author": "cpuabuse.com",
-	"license": "ISC",
-	"bugs": {
-		"url": "https://github.com/cpuabuse/cross-cat/issues"
-	},
-	"homepage": "https://cpuabuse.github.io/cross-cat",
-	"dependencies": {
-		"commander": "^4.1.1",
-		"get-stdin": "^7.0.0"
-	},
-	"devDependencies": {
-		"@istanbuljs/nyc-config-typescript": "^1.0.1",
-		"@types/mocha": "^5.2.7",
-		"@types/node": "^13.9.8",
-		"@typescript-eslint/eslint-plugin": "^2.26.0",
-		"@typescript-eslint/parser": "^2.26.0",
-		"chirpchirp": "0.0.6",
-		"codacy-coverage": "^3.4.0",
-		"eslint": "^6.8.0",
-		"eslint-config-airbnb-base": "^14.1.0",
-		"eslint-config-prettier": "^6.10.1",
-		"eslint-plugin-import": "^2.20.2",
-		"eslint-plugin-mocha": "^6.3.0",
-		"eslint-plugin-prettier": "^3.1.3",
-		"mocha": "^7.0.1",
-		"mocha-junit-reporter": "^1.23.3",
-		"nyc": "^15.0.0",
-		"prettier": "^2.0.4",
-		"release-it": "^13.5.1",
-		"rollup": "^2.3.2",
-		"ts-node": "^8.6.2",
-		"typedoc": "^0.16.11",
-		"typescript": "^3.7.3"
-	}
-}
+{
+	"name": "cross-cat",
+	"version": "1.0.0",
+	"description": "A cross platform cat command",
+	"main": "src/cross-cat.js",
+	"bin": {
+		"cat": "src/cross-cat.js",
+		"cross-cat": "src/cross-cat.js"
+	},
+	"directories": {
+		"test": "test"
+	},
+	"engines": {
+		"node": ">=10"
+	},
+	"types": "src/cross-cat.d.ts",
+	"scripts": {
+		"build": "tsc",
+		"build:docs": "typedoc",
+		"coverage:cobertura": "nyc --reporter cobertura npm run test:ts:mocha",
+		"coverage:lcov": "nyc npm run test:ts:mocha",
+		"coverage:codacy": "npm run coverage:lcov && npx cross-cat coverage/lcov.info | codacy-coverage",
+		"test:js:junit": "npm run build && mocha test --reporter mocha-junit-reporter",
+		"test:js:mocha": "npm run build && mocha",
+		"test:ts:mocha": "npm run build && mocha --require ts-node/register test/test.ts",
+		"abc": "npx chirpchirp send --message \"abc abc\n abc abc\""
+	},
+	"repository": "github:cpuabuse/cross-cat",
+	"keywords": [
+		"cat",
+		"cli",
+		"concat"
+	],
+	"author": "cpuabuse.com",
+	"license": "ISC",
+	"bugs": {
+		"url": "https://github.com/cpuabuse/cross-cat/issues"
+	},
+	"homepage": "https://cpuabuse.github.io/cross-cat",
+	"dependencies": {
+		"commander": "^4.1.1",
+		"get-stdin": "^7.0.0"
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Format Error: unexpected "\r"](https://app.codacy.com/gh/cpuabuse/cross-cat/pullRequest?prid=5354161)

Reply:
Not needed.